### PR TITLE
Initial Implementation

### PR DESCRIPTION
**Pull request by andrewhowdencom**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+## The port that thumbor runs on
+## (Optional)
+# thumbor_port: "8888"
+
+## The IP that thumbor is bound to
+## (Optional)
+# thumbor_ip: "127.0.0.1"
```
Comment (PatrickKubica) [inline defaults/main.yml]: can we add a newline here?

File: tasks/dependencies.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+- name: "Ensure required dependencies are present"
+  package:
+    name: "{{ item }}"
+    state: "present"
+  with_items:
+    # python is python2
+    - "python"
+    - "python-pip"
+    - "libcurl4-openssl-dev"
```
Comment (PatrickKubica) [inline tasks/dependencies.yml]: can we add a newline here?

File: templates/etc/default/thumbor.j2 [hunk 3]
```diff
@@ -0,0 +1,2 @@
+THUMBOR_PORT="{{ thumbor_port | default('8888') }}"
+THUMBOR_IP="{{ thumbor_ip | default('127.0.0.1') }}"
```
Comment (PatrickKubica) [inline templates/etc/default/thumbor.j2]: can we add a newline here as well please so we are consistent?

**Comment by andrewhowdencom** [comment]
I may or may not have spent 10 minutes trying to work out (unsuccessfully) how to script this

**Pull request merged**